Lib-DNDBeyond/dndb_CastSpell_processor.command

```diff
@@ -0,0 +1,51 @@
+[h: log.debug (getMacroName() + ": args = " + json.indent (macro.args))]
+[h: actionButton = json.get (macro.args, "actionButton")]
+[h: spellSlot = json.get (macro.args, "spell-slot")]
+[h, if (spellSlot == ""): spellSlot = 0; ""]
+[h: castSpellAdvantage = json.get (macro.args, "castSpell-advantage")]
+[h: castSpellDisadvantage = json.get (macro.args, "castSpell-disadvatage")]
+[h: castSpellBoth = json.get (macro.args, "castSpell-both")]
+[h: spellName = json.get (macro.args, "spellName")]
```

Comment:
refactor to reflect that this value is encoded, aka 'encodedSpellName'

Reply:
In fact, this all looks very familiar. I think SpellEditor opens with the same pattern. Go ahead and stop repeating code and see if this can be encapsulated somewhere.